src/golang/cmd/server/handler/test_integration.go

```diff
@@ -0,0 +1,88 @@
+package handler
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/aqueducthq/aqueduct/cmd/server/routes"
+	"github.com/aqueducthq/aqueduct/lib/collections/integration"
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	aq_context "github.com/aqueducthq/aqueduct/lib/context"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/job"
+	"github.com/aqueducthq/aqueduct/lib/vault"
+	"github.com/aqueducthq/aqueduct/lib/workflow/operator/connector/auth"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+)
+
+// TestIntegrationHandler tries to connect to an existing integration.
```

Comment:
We may want to go through all the handlers, including this, and add a comment with the expected input/output like in https://github.com/aqueducthq/aqueduct/blob/1cb2e166caf6793573e1f8206bfa285416413674/src/golang/cmd/server/handler/preview_table.go#L30-L40.